doc/content/tutorials/KRUSTY.md

```diff
@@ -0,0 +1,885 @@
+
+!listing /tutorials/krusty/KRUSTY/solide.i
+  block=Kernels
+  
+Next, the boundary conditions on the solid are applied. In case A, coupling only OpenMC and MOOSE Heat Transfer Model, the boundary conditions on each heat pipe will be the Dirichlet type, where they have a constant temperature of $1073 K$ and for the "outside" boundary conditions will be a [ConvectiveHeatFluxBC](https://mooseframework.inl.gov/source/bcs/ConvectiveHeatFluxBC.html) type, where it computes the convective heat flux, with the convective heat transfer coefficient and the far field temperature coupled as material properties.
+
+!listing /tutorials/krusty/KRUSTY/solide.i
+  block=BCs
+
+The [Transfer](https://mooseframework.inl.gov/syntax/Transfers/index.html) system in MOOSE is used to communicate variables across applications; a heat source will be computed by OpenMC and applied as a source term in MOOSE. In the opposite direction, MOOSE will compute a temperature that will be applied to the OpenMC geometry. We need to add an auxiliary variable to receive the heat source from OpenMC, which is defined as `[power]`.
+
+
+!listing /tutorials/krusty/KRUSTY/solide.i
+  block=AuxVariables
+
+We also set thermal conductivity values in the blocks: fuel, `cavity_center`, `gap_clamp`, `gap_ref`, `gap_sleeve`, `gap_vaccan`. Constant values are used for simplicity.
+
+!listing /tutorials/krusty/KRUSTY/solide.i
+  block=Materials
+
+
+By setting a fixed number of time steps, `dt`, this example will simply run a fixed number of Picard iterations.
+
+!listing /tutorials/krusty/KRUSTY/solide.i
+  block=Executioner
+
+MOOSE will provide the solid temperature for OpenMC so we add postprocessors to evaluate the average, minimum and maximum temperature in the fuel. 
+
+!listing /tutorials/krusty/KRUSTY/solide.i
+  start=Postprocessors
+
+### Neutronics Input Files
+
+We begin by defining a mesh on which OpenMC will receive temperature from the couple MOOSE application.
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  end=AuxVariables
+
+Then, we can define our initial conditions for the temperature, which will be a constant function of $1073 K$
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  start=ICs
+  end=Problem
+
+Next, we define the `[Problem]` block, which will describe all objects necessary for the physics solve. In order to replace MOOSE finite element calculations with OpenMC particle transport calculations, we are going to use the [OpenMCCellAverageProblem](https://cardinal.cels.anl.gov/source/problems/OpenMCCellAverageProblem.html) class. In this model, we specify the total fission power by which to normalize OpenMC's tally results. Next, we indicate which blocks in the `[Mesh]` should be considered for temperature feedback using the `temperature_blocks` parameter. Mesh tallies were used to estimate the heat source, as indicated by `tally_type`, where the same mesh can be translated to multiple locations in the OpenMC geometry, specifying a mesh file using `mesh_template`.
```

Comment:
I'd omit the part about the `mesh_template`, since that's not used in the tutorial